starter-core/src/main/java/io/micronaut/starter/feature/agorapulse/worker/Worker.java

```diff
@@ -0,0 +1,216 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.agorapulse.worker;
+
+import com.fizzed.rocker.RockerModel;
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.agorapulse.AgoraPulseFeature;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestDistributedJobGroovy;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestDistributedJobJava;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestDistributedJobKotlin;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestDistributedJobSpecGroovy;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestDistributedJobTestJava;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestDistributedJobTestKotlin;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestDistributedJobTestKotest;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestSimpleJobGroovy;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestSimpleJobJava;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestSimpleJobKotlin;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestSimpleJobSpecGroovy;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestSimpleJobTestJava;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestSimpleJobTestKotlin;
+import io.micronaut.starter.feature.agorapulse.worker.template.emailDigestSimpleJobTestKotest;
+import io.micronaut.starter.options.Language;
+import io.micronaut.starter.options.TestFramework;
+import io.micronaut.starter.template.RockerTemplate;
+import jakarta.inject.Singleton;
+
+import java.util.Optional;
+
+
+@Singleton
+public class Worker implements AgoraPulseFeature {
+
+    private static final String ARTIFACT_ID = "micronaut-worker";
+    private static final String AWAITILITY_ARTIFACT_ID = "awaitility";
+
+    @Override
+    @NonNull
+    public String getName() {
+        return "micronaut-worker";
+    }
+
+    @Override
+    public String getTitle() {
+        return "Advanced distributed scheduling for Micronaut";
+    }
+
+    @Override
+    public boolean supports(ApplicationType applicationType) {
+        return applicationType == ApplicationType.DEFAULT;
+    }
+
+    @Override
+    public String getCategory() {
```

Comment:
I don't think other is a valid category.

Reply:
but which one is the best?

Reply:
🤷